NSS-Day-Cohort-68/rare-client-rare-team3
# Delete a post

I created a delete button for posts belonging to the logged in user. This allows the user to delete his/her own posts. This completes ticket #35

To test:
1. `git fetch --all` 
2. switch to this branch `stacy/posts/delete_post/client`
3. `npm start` in client repository to bring up application in browser and make sure the api is started
4. If not already logged in, login as a user from your database
5. Click on the `All Posts` button in the nav bar
6. Test the delete functionality by clicking on the trash icon on the current logged in user's post
7. Clicking 'cancel' should do nothing to the list/database and clicking 'ok' should remove the post from list/database
8. Verify that the post was removed from the list
9. Repeat the same process to test the delete functionality within the `My Posts` view.
